bundles/aero.minova.workingtime.helper/src/aero/minova/workingtime/wizard/PeriodPage.java

```diff
@@ -0,0 +1,291 @@
+package aero.minova.workingtime.wizard;
```

Comment:
Diese Klasse gehört NICHT in dieses Repository!
Sie gehört zu dem workingtime Helper und muss entsprechend in aero.minova.workingtime!

Wenn dort ein Update nötig ist muss auch ein neuer Release erstellt werden